website/content/v1.7/reference/configuration/v1alpha1/config.md

```diff
@@ -1558,32 +1558,38 @@ DeviceWireguardConfig contains settings for configuring Wireguard network interf
 machine:
     network:
         interfaces:
-            - wireguard:
-                privateKey: ABCDEF... # Specifies a private key configuration (base64 encoded).
-                listenPort: 51111 # Specifies a device's listening port.
-                # Specifies a list of peer configurations to apply to a device.
-                peers:
-                    - publicKey: ABCDEF... # Specifies the public key of this peer.
-                      endpoint: 192.168.1.3 # Specifies the endpoint of this peer entry.
-                      # AllowedIPs specifies a list of allowed IP addresses in CIDR notation for this peer.
-                      allowedIPs:
-                        - 192.168.1.0/24
+          - interface: wg0 # Name of the wireguard interface
```

Comment:
this is a good change, but this part of the documentation is automatically generated, so we'd need to figure out how to incorporate that into the doc generator (and probably in other similar places)